src/sql/util.py

```diff
@@ -420,6 +420,15 @@ def get_line_content_from_toml(file_path, line_number):
         return eline, ekey, evalue
 
 
+def to_upper_if_snowflake_conn(conn, upper):
+    return (
+        upper.upper()
+        if isinstance(conn, AbstractConnection)
```

Comment:
do we need to check if this is an abstract connection? do anything fails if we don't?

Reply:
Just wanna make sure there is callable function we can invoke
fix in [4a4f218](https://github.com/ploomber/jupysql/pull/697/commits/4a4f218bd3fbf7a533926f9a1f69d6fb2026d6c7)